apollo-federation/src/operation/rebase.rs

```diff
@@ -241,7 +241,10 @@ impl FieldSelection {
             return Ok(Some(Selection::from(self.clone())));
         }
 
-        let Some(rebased) = self.field.rebase_on(parent_type, schema, error_handling)? else {
+        let Some(rebased) =
+            self.field
+                .rebase_on(parent_type, schema, RebaseErrorHandlingOption::IgnoreError)?
```

Comment:
this is incorrect -> we should pass the original `error_handling` option -> if we cannot rebase and error handling is set to ignore we return `None` otherwise we return an error (see https://github.com/apollographql/federation/blob/main/internals-js/src/operations.ts#L3096)